profile/perf_test.go

```diff
@@ -0,0 +1,122 @@
+package profile
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+type testCase struct {
+	warmTime, tearDownTime float64
+	expected               []map[string]float64
+}
+
```

Comment:
please add a test for testing PerfStat.Run() function

Reply:
Done